addons/binding/org.openhab.binding.milightdev/src/main/java/org/openhab/binding/milight/internal/simulatedbridge/TestDiscovery.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.milight.internal.simulatedbridge;
+
+import org.eclipse.smarthome.config.discovery.AbstractDiscoveryService;
+import org.openhab.binding.milight.MilightBindingConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A simple discovery service that will make the emulated V6 bridge visible to the Inbox of OH.
+ * Enable this in OSGI-INF/TestDiscovery.xml with enabled="true".
+ *
+ * @author David Graeff
+ * @since 2.1
+ */
+public class TestDiscovery extends AbstractDiscoveryService {
+    private final Logger logger = LoggerFactory.getLogger(TestDiscovery.class);
+
+    @SuppressWarnings("unused")
+    private EmulatedV6Bridge server;
+
+    public TestDiscovery() {
+        super(MilightBindingConstants.BRIDGE_THING_TYPES_UIDS, 2, true);
+        try {
+            server = new EmulatedV6Bridge();
+        } catch (Exception e) {
+            logger.warn("An error occurred", e);
+        }
+    }
+
+    @Override
+    protected void startScan() {
+    }
+}
```

Comment:
It seems you did not commit the actual removal, now they are just copies, see also:

https://github.com/davidgraeff/openhab2-addons/tree/milightfix/addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/test